scripts/dqlite/Makefile

```diff
@@ -1,6 +1,3 @@
-SHELL = bash
-.ONESHELL:
```

Comment:
This was a wart, that allowed us to prototype really easily, but isn't great long term.